dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/TaskMainInfo.java

```diff
@@ -65,12 +65,12 @@ public class TaskMainInfo {
     /**
      * processDefinitionCode
      */
-    private long processDefinitionCode;
+    private Long processDefinitionCode;
```

Comment:
Hi @jackfanwan I think this change can not resolve the issue. 

If delete workflow, the related task definitions should be deleted too.